types/react-flatpickr/index.d.ts

```diff
@@ -4,13 +4,19 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.3
 
-import { Component } from 'react';
-import { EventCallback, Options } from 'flatpickr';
+import { Component, DetailedHTMLProps, InputHTMLAttributes, HTMLAttributes } from 'react';
+import { Hook, Options } from 'flatpickr';
 
-export interface DateTimePickerProps {
+type HTMLProps =
```

Comment:
You either need to inline this type below, or export it (because if you run `npm run lint react-flatpickr`, you get a lint error for this).